### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by browner12**
- `align` is an obsolete HTML5 attribute. replace with CSS styling.
- add an `alt` attribute to the `` tag for accessibility.
- remove redundant `</img>` closing tag.
- the `<center>` tag is not supported in HTML5. replace with CSS styling.
- the `<bold>` tag does not exist. The `<h4>` receives bolding anyway, so the whole line is bolded.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></img></a>
+<p style="text-align: center; ">
+  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></a>
   <a href="https://packagist.org/packages/laravel-zero/framework"><img src="https://img.shields.io/packagist/dt/laravel-zero/framework.svg" alt="Total Downloads"></a>
   <a href="https://packagist.org/packages/laravel-zero/framework"><img src="https://img.shields.io/packagist/v/laravel-zero/framework.svg?label=stable" alt="Latest Stable Version"></a>
   <a href="https://packagist.org/packages/laravel-zero/framework"><img src="https://img.shields.io/packagist/l/laravel-zero/framework.svg" alt="License"></a>
 </p>
 
-<h4> <center>This is a <bold>community project</bold> and not an official Laravel one </center></h4>
+<h4 style="text-align: center; ">This is a community project and not an official Laravel one</h4>
```
Comment (owenvoke) [inline README.md:12]: Can you completely remove this line? 🤷🏻 It's true, but Nuno removed it from the main repository (so best to keep them in sync).

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
-<p align="center">
-    <img title="Laravel Zero" height="100" src="https://raw.githubusercontent.com/laravel-zero/docs/master/images/logo/laravel-zero-readme.png" />
+<p style="text-align: center; ">
+    <img title="Laravel Zero" height="100" src="https://raw.githubusercontent.com/laravel-zero/docs/master/images/logo/laravel-zero-readme.png" alt="Laravel Zero Logo" />
 </p>
 
-<p align="center">
-  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></img></a>
+<p style="text-align: center; ">
+  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"></a>
```
Comment (owenvoke) [inline README.md:6]: Although it does work to not have a closing tag, it's probably good to keep a self-closing tag.
```suggestion
  <a href="https://github.com/laravel-zero/framework/actions"><img src="https://img.shields.io/github/actions/workflow/status/laravel-zero/framework/tests.yml?branch=master" alt="Build Status"/></a>
```

**Review by owenvoke — CHANGES_REQUESTED** [review]

**Comment by owenvoke** [comment]
The `style` attribute (and CSS in general) does not work on GitHub, which is why the `align` tag is used. This is not ideal (and I agree that it should be removed in future), but it works. 🤷🏻

File: README.md (hunk 2 above)
Comment (browner12) [reply README.md:6]: I'm cool with that if you guys are. I prefer the self closing tag, but I was trying to keep it consistent with the following `` tags.

File: README.md (hunk 1 above)
Comment (browner12) [reply README.md:12]: I'm good with that if you guys are.

**Comment by browner12** [comment]
All requested changes should be made. LMK if you'd like anything else.

**Review by owenvoke — APPROVED** [review]
Thank you!

**Pull request merged**